Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -25,9 +25,13 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 {
     batteryStateOfChargeService_.addData(batteryData);
 
+
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
-}
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed();
+           if(batteryStateOfChargeService_.isCharging()){
+              QTextStream(stdout)<< " Time Until Charged: " <<batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString("hh:mm:ss")<<endl;
```

Comment:
Place a space between each operator:
`QTextStream(stdout) << "Time until charged: "`

Reply:
Also, the time should be in the same format as handled in LogFileReader